tom_nonsidereal_airmass/templatetags/nonsidereal_airmass_extras.py

```diff
@@ -1,10 +1,19 @@
 from django import template
+from django.conf import settings
 from dateutil.parser import parse
 import plotly.graph_objs as go
 from plotly import offline
 
-from tom_nonsidereal_airmass.utils import get_visibility
+from tom_nonsidereal_airmass.utils import get_visibility, get_arc
 from tom_nonsidereal_airmass.forms import NonsiderealTargetVisibilityForm
+from tom_targets.models import Target
+
+import json
+import numpy as np
+
+USE_EPHEMERIS_SCHEME = getattr(settings, "USE_EPHEMERIS_SCHEME", None)
+if USE_EPHEMERIS_SCHEME:
+    from tom_ephemeris_airmass.utils import get_eph_scheme_data
```

Comment:
We're obviously having this conversation in Slack right now, but just to note that this dependency should be added to `setup.py`.